docs/docs/api/promise.on.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: api
+id: promise.on
+title: Promise.on
+---
+
+
+[← Back To API Reference](/docs/api-reference.html)
+<div class="api-code-section"><markdown>
+##Promise.on
+
+```js
+Promise.on(String eventName("created"|"chained"|"fulfilled"|"rejected"|"following"|"cancelled"), Function handler, [Boolean simpleEventAPI]) -> undefined
```

Comment:
@lextiz but that promise is created by library's bluebird instance. Therefore, the promise derived with `then` will also be created by that instance, and as such will not be tracked. You would have to use Promise.resolve(db.query(...)).then(...) instead of simply chaining `then`, and you'd have to do it every time you use an external library.